src/Aspire.Dashboard/Extensions/ResourceViewModelExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Dashboard.Model;
+
+namespace Aspire.Dashboard.Extensions;
+
+public static class ResourceViewModelExtensions
```

Comment:
Looks like all members are internal, so consider making the class internal too (or even instead).

Reply:
Good call, thank you. I will make this change